android/src/main/java/com/cuhacking/atlas/search/SearchResultsAdapter.kt

```diff
@@ -0,0 +1,49 @@
+package com.cuhacking.atlas.search
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.cuhacking.atlas.R
+
+class SearchResultsViewHolder(view: View) : RecyclerView.ViewHolder(view) {
+    private val result: TextView = view.findViewById(R.id.result)
+    private val description: TextView = view.findViewById(R.id.description)
+
+    fun bind(searchResult: SearchResult) {
+        result.text = searchResult.name
+        description.text = searchResult.description
+    }
+}
```

Comment:
You can also use view binding here!

```suggestion
class SearchResultsViewHolder(private val binding: ItemSearchResultBinding) : RecyclerView.ViewHolder(binding.root) {
    fun bind(searchResult: SearchResult) {
        binding.result.text = searchResult.name
        binding.description.text = searchResult.description
    }
}
```